src/components/screens/public/ConsentProvider/ConsentProviderController.js

```diff
@@ -68,6 +68,7 @@ class ConsentProviderController extends VuexModule {
       requested_scope: scopesList,
       skip: isSkip,
       redirect_url: redirectUrl,
+      client_name: appName,
```

Comment:
![image](https://user-images.githubusercontent.com/756731/85853448-31c54280-b7bb-11ea-8c29-1bd7932a88a6.png)

no any app name in description.
structure from server have other fields, you should get `client` at first, then get `client_name`
```
    const {
      requested_scope: scopesList,
      skip: isSkip,
      redirect_url: redirectUrl,
      client,
    } = consentDetails;
    const { client_name: appName } = client;
```
and after that you will receive this

![image](https://user-images.githubusercontent.com/756731/85853523-53262e80-b7bb-11ea-8def-5f81c657eb19.png)

and i agree with Danil to make maybe strong/bold app name here, for underline that this is the app name